pipelines_codefresh.yml

```diff
@@ -0,0 +1,14 @@
+version: '1.0'
```

Comment:
Move this file to `codefresh/terraform.yml`

Reply:
done